docker-compose-dev.yml

```diff
@@ -39,9 +39,9 @@ services:
       - dc
     volumes:
       - ./sdc-data:/sdc-data
-      - ./agent-data:/data
       - dummy-destination-data:/output
       - ./agent:/usr/src/app
+      - ./data:/usr/src/app/data
```

Comment:
Then in `Dockerfile.local` specify `DATA_DIR=/usr/src/app/data`

Reply:
as I reverted DATA_DIR to "data" we don't need it